File: ojuz/ioi/2020/Day2/mushrooms/java/mushrooms.java
public class mushrooms {

	public int count_mushrooms(int n) {
		int[] m = new int[n];
		for (int i = 0; i < n; i++)
			m[i] = i;
		int c1 = stub.use_machine(m);
		m = new int[] {0, 1};
		int c2 = stub.use_machine(m);
		return c1+c2;
	}
}

File: ojuz/ioi/2020/Day2/mushrooms/py/mushrooms.py
from stub import use_machine

def count_mushrooms(n: int) -> int:
	m = []
	for i in range(n):
		m.append(i)
	c1: int = use_machine(m)
	m = [0, 1]
	c2: int = use_machine(m)
	return c1+c2
